app/models/user.rb

```diff
@@ -0,0 +1,4 @@
+class User < ApplicationRecord
+  validates :fullname, presence: true
```

Comment:
Does Devise already require a password or do we need to validate it here?

Reply:
Devise already requires a password